src/stories/Introduction.stories.mdx

```diff
@@ -1,14 +1,14 @@
 import { Meta } from '@storybook/addon-docs';
-import Code from './assets/code-brackets.svg';
-import Colors from './assets/colors.svg';
-import Comments from './assets/comments.svg';
-import Direction from './assets/direction.svg';
-import Flow from './assets/flow.svg';
-import Plugin from './assets/plugin.svg';
-import Repo from './assets/repo.svg';
-import StackAlt from './assets/stackalt.svg';
+import Code from './assets/code-brackets.svg?raw';
+import Colors from './assets/colors.svg?raw';
+import Comments from './assets/comments.svg?raw';
+import Direction from './assets/direction.svg?raw';
+import Flow from './assets/flow.svg?raw';
+import Plugin from './assets/plugin.svg?raw';
+import Repo from './assets/repo.svg?raw';
+import StackAlt from './assets/stackalt.svg?raw';
```

Comment:
For SVG imports outside of node_modules, `?raw` results in icons not being rendered, but `?url` works for me.

And actually, with the added `defaultImport: 'url'` addition to the `vite.config.ts` below, we don't need to specify the import type here.
 
 http://localhost:6006/?path=/docs/example-introduction--docs

|?raw|?url|
|---|---|
|<img width="509" alt="Screen Shot 2023-05-11 at 4 10 29 PM" src="https://github.com/cfpb/design-stories/assets/2592907/c64a0d12-2d6b-484c-aca6-6107a6bdffc7">|<img width="522" alt="Screen Shot 2023-05-11 at 4 10 39 PM" src="https://github.com/cfpb/design-stories/assets/2592907/34f67e0b-c587-4616-b8a0-2ca358f557a0">|

```suggestion
import Code from './assets/code-brackets.svg';
import Colors from './assets/colors.svg';
import Comments from './assets/comments.svg';
import Direction from './assets/direction.svg';
import Flow from './assets/flow.svg';
import Plugin from './assets/plugin.svg';
import Repo from './assets/repo.svg';
import StackAlt from './assets/stackalt.svg';
```